doc/book/replication/repl_leader_elect.rst

```diff
@@ -108,6 +108,18 @@ When the fencing is on, the leader resigns its leadership if it has less than th
 of alive connections to the cluster nodes. The resigning leader receives the status of a follower in the current election term and becomes read-only.
 Fencing applies to the instances that have the :ref:`election_mode <repl_leader_elect_config>` set to "candidate" or "manual".
 
+.. _repl_leader_elect_splitbrain:
+
+There can still be a situation when a replica set has two leaders working independently (so called *split-brain*).
+It can happen, for example, if a user mistakenly lowered the :ref:`replication_synchro_quorum <repl_leader_elect_config>` below ``N / 2 + 1``.
+In this situation, to preserve the data integrity, if an instance detects the split-brain anomaly in the incoming replication data,
+it breaks the connection with the instance sending the data and writes the ``ER_SPLIT_BRAIN`` error in the log.
+
+Once noticing this error, a user has to manually inspect the data on both instances having the leader status,
```

Comment:
It's not necessary that any nodes in the split halves will be leaders.

The user will see that there are two sets of nodes, which do not intersect, and any node from one set is disconnected from any node from the other set with ER_SPLIT_BRAIN error.

Then the user might change any representative from each of the sets, and inspect the data on it.

The sets essentially represent an old leader and all the nodes which followed that leader and believe that it's the source of truth.

Reply:
Re-written. Please take a look.